Runar/Runar/Controllers/Misc/Processing/ProcessingVC.swift

```diff
@@ -342,6 +341,11 @@ class ProcessingVC: UIViewController {
         ])
     }
     
+    private func configureNavigationBar() {
+        navigationItem.hidesBackButton = true
```

Comment:
Please, rename this property hidesBackButton. Apply, please, needToHideBackButton, for example.